operating/manage-vulnerabilities/common-vuln-management-tasks.adoc

```diff
@@ -1,45 +1,82 @@
-Following are some common tasks you can perform from the *Vulnerability Management* -> *Dashboard* view.
 
-include::modules/find-critical-cves-impacting-your-infrastructure.adoc[leveloffset=+1]
+[id="viewing-vulnerabilities"]
+== Viewing vulnerabilities
 
-include::modules/find-the-most-vulnerable-image-components.adoc[leveloffset=+1]
+Historically, {product-title-short} provided a view of vulnerabilities discovered in your system in the vulnerability management dashboard. The dashboard is deprecated in {product-title-short} 4.5 and will be removed in a future release. For more information about the dashboard, see xref:../../operating/manage-vulnerabilities/vulnerability-management-dashboard.adoc#vulnerability-management-dashboard[Using the vulnerability management dashboard].
 
-include::modules/identify-container-image-layer-that-introduces-vulnerabilities.adoc[leveloffset=+1]
+The *Vulnerability Management* -> *Workload CVEs* page provides information about vulnerabilities in applications running on clusters in your system. You can view vulnerability information across images and deployments. The *Workload CVEs* page provides advanced filtering capabilities, including the ability to view images and deployments with vulnerabilities and filter by image, deployment, namespace, cluster, CVE, component, and component source.
 
-include::modules/identify-dockerfile-line-component-cve.adoc[leveloffset=+1]
+//Viewing CVEs
+include::modules/vulnerability-management20-view-cve.adoc[leveloffset=+1]
+
+//Viewing vulns in nodes
+include::modules/identify-vulnerabilities-in-nodes-vm20.adoc[leveloffset=+1]
+include::modules/disable-identify-vulnerabilities-in-nodes.adoc[leveloffset=+2]
+.Additional resources
+* xref:../../operating/examine-images-for-vulnerabilities.adoc#supported-operating-systems_examine-images-for-vulnerabilities[Supported operating systems]
+
+//viewing vulns in platforms
+
+include::modules/vulnerability-management20-view-platform-cve.adoc[leveloffset=+1]
+
+[id="excluding-CVEs"]
+== Excluding CVEs
 
-include::modules/view-details-only-for-fixable-cves.adoc[leveloffset=+1]
+You can exclude or ignore CVEs in {product-title-short} by snoozing node and platform CVEs and deferring or marking node, platform, and image CVEs as false positives. You might want to exclude CVEs if you know that the CVE is a false positive or you have already taken steps to mitigate the CVE. Snoozed CVEs do not appear in vulnerability reports or trigger policy violations.
 
-include::modules/identify-operating-system-of-the-base-image.adoc[leveloffset=+1]
+You can snooze a CVE to globally ignore it for a specified period of time. Snoozing a CVE does not require approval.
 
-include::modules/identify-top-risky-objects.adoc[leveloffset=+1]
+[NOTE]
+====
+Snoozing node and platform CVEs requires enabling the ROX_VULN_MGMT_LEGACY_SNOOZE environment variable.
```

Comment:
```suggestion
To snooze node and platform CVEs, you need to enable the `ROX_VULN_MGMT_LEGACY_SNOOZE` environment variable.
```

Reply:
The person snoozing CVEs is not necessarily the person enabling this feature, so I think it is more accurate to say that it requires the variable to be enabled.